pyproject.toml

```diff
@@ -7,3 +7,57 @@ disable_error_code = ["union-attr", "annotation-unchecked", "import-untyped"]
 explicit_package_bases = true
 warn_unused_ignores = true
 exclude = ['tests']
+
+[tool.poetry]
+name = "tagstudio"
+version = "9.3.2"
+packages = [
+  {include = "tagstudio/tag_studio.py"},
+  {include = "tagstudio/src/core"},
+  {include = "tagstudio/src/qt"},
+  {include = "tagstudio/resources"},
+  {include = "tagstudio/__init__.py"}
+]
+
+description = "User-Focused Photo & File Management System"
+readme = "README.md"
+authors = ["Travis Abendshien"]
+license = "GPL-3.0-only"
+repository = "https://github.com/TagStudioDev/TagStudio"
+maintainers = [] # Maybe someone can find a way to export this from GitHub?
+
+[tool.poetry.urls]
+Discord = "https://discord.gg/hRNnVKhF2G"
+
+[tool.poetry.dependencies]
```

Comment:
Dependencies should use `-` instead of `_`
eg: `opencv_python`  -> `opencv-python`